schemas/vnsutils.json

```diff
@@ -181,42 +181,49 @@
                 "sectionEnd": "vCenter Parameters",
                 "propertyOrder": 190
             },
+            "vcenter_vmfolder": {
+                "type": "string",
```

Comment:
There is no default value provided. By default, any component parameters will default to the equivalent global parameter in common.yml. Notice line 178 in this file. This property should have a similar `default` line, something like:

"default": "(global vCenter VM folder)"